### PR TITLE
Update to Ember v3.25

### PR DESCRIPTION
**Pull request by bertdeblock**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -10,11 +10,14 @@
 /.editorconfig
 /.ember-cli
 /.env*
+/.eslintcache
 /.eslintignore
 /.eslintrc.js
 /.git/
 /.github/
 /.gitignore
+/.prettierignore
```
Comment (bertdeblock) [inline .npmignore:19]: These two are not yet part of the latest blueprint but a PR for this was merged recently:
https://github.com/ember-cli/ember-cli/pull/9437

File: .npmignore [hunk 2]
```diff
@@ -24,6 +27,7 @@
 /ember-cli-build.js
 /testem.js
 /tests/
+/yarn-error.log
```
Comment (bertdeblock) [inline .npmignore:30]: Same goes for this one: https://github.com/ember-cli/ember-cli/pull/9453

File: addon/helpers/page-title.js [hunk 3]
```diff
@@ -17,8 +17,8 @@ export default class PageTitle extends Helper {
     return guidFor(this);
   }
 
-  init() {
-    super.init(...arguments);
+  constructor() {
```
Comment (bertdeblock) [inline addon/helpers/page-title.js:20]: Lint fix.

File: addon/services/page-title-list.js [hunk 4]
```diff
@@ -37,11 +37,11 @@ export default class PageTitleListService extends Service {
     prepend: true,
 
     // The default replace value to use.
-    replace: null
+    replace: null,
   };
 
-  init() {
-    super.init();
+  constructor() {
```
Comment (bertdeblock) [inline addon/services/page-title-list.js:43]: Lint fix.

File: tests/dummy/config/targets.js [hunk 5]
```diff
@@ -13,6 +13,7 @@ if (isCI || isProduction) {
   browsers.push('ie 11');
 }
 
-module.exports = {
-  browsers
-};
+module.exports = [
```
Comment (bertdeblock) [inline tests/dummy/config/targets.js:16]: Temp fix for: https://github.com/emberjs/ember.js/issues/19353

Comment (bertdeblock) [reply tests/dummy/config/targets.js:16]: We don't need to merge it like this, but I already did the work for this PR so wanted to create it anyways.

**Comment by raido** [comment]
Could check what's up with Embroider test run?

**Comment by bertdeblock** [comment]
Hi @raido, yup, I haven't time yet to properly check it. Do you by chance have any idea? I'm not familiar with embroider, so I don't see it right away. I might have another look at it over the weekend.

**Comment by bertdeblock** [comment]
Can't seem to reproduce it. Locally the `embroider-tests` scenario passes.

File: tests/dummy/config/targets.js (hunk 5 above)
Comment (bertdeblock) [reply tests/dummy/config/targets.js]: This is now fixed in `ember-source`.

File: package.json [hunk 6]
```diff
@@ -17,55 +17,62 @@
...
+    "eslint": "^7.20.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-ember": "^10.2.0",
     "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.1.2",
+    "prettier": "^2.2.1",
+    "qunit": "2.13.0",
```
Comment (bertdeblock) [inline package.json:75]: Temp pin `ember-qunit` and `qunit`. More info in the related commit and [here on Discord](https://discord.com/channels/480462759797063690/568935504288940056/818524713495429140). Will create a new PR to unpin again when fixed in `qunit` (or we can wait with merging of course until the fix has landed).

**Comment by bertdeblock** [comment]
@raido All checks are okay now.

File: package.json [hunk 7]
```diff
@@ -17,55 +17,62 @@
...
-    "ember-cli-terser": "^4.0.0",
+    "ember-cli-terser": "^4.0.1",
     "ember-cli-update": "^0.54.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-inline-svg": "^1.0.0",
-    "ember-load-initializers": "^2.1.1",
+    "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.6.0",
+    "ember-qunit": "5.1.1",
```
Comment (rwjblue) [inline package.json:60]: This can be updated now that 5.1.4 is out (includes https://github.com/emberjs/ember-qunit/pull/827)

Comment (bertdeblock) [reply package.json]: Updated. Thanks for notifying!

**Comment by rwjblue** [comment]
I think they have crashed, they aren't running that long (an error probably happened early and the test harness didn't detect it so it has "hung" the job).

**Comment by rwjblue** [comment]
Add qunit to skipBabel config for embroider, it should pass then. (Either that or remove IE11 support, but not sure if that is something we want to do now).

**Review by raido — CHANGES_REQUESTED** [review]
Let's update ember-try scenarios for 3.24 LTS + make CI to run it :)

**Comment by bertdeblock** [comment]
Done! I didn't notice the LTS scenario was missing. [Created a PR](https://github.com/ember-cli/ember-cli/pull/9485) to add it to the addon blueprint.

**Review by raido — APPROVED** [review]

**Comment by raido** [comment]
@bertdeblock Thanks you!

**Comment by raido** [comment]
Documentation page runs on 3.25.x now as well.

**Pull request merged**